boards/feather-nrf52840-sense/Makefile.dep

```diff
@@ -0,0 +1,11 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += apds9960
+  USEMODULE += bmp280_i2c
+  USEMODULE += lis3mdl
```

Comment:
didn't you write the LSM6DS33 does not work with this driver?

Reply:
Ah nvm lis3mdl is a magnetometer

Reply:
Yeah but LSM6DS33 != LIS3MDL ;) The existing `lsm6dsl` driver can probably be adjusted for the LSM6DS33 (as per the issue I linked above). But I guess that is out of scope for this PR.